packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -372,10 +392,23 @@ const _BaseButton: React.ForwardRefRenderFunction<BladeElementRef, BaseButtonPro
     hasIcon: Boolean(Icon),
   });
 
+  const renderElement = React.useMemo(() => getRenderElement(href), [href]);
+
   return (
     <StyledBaseButton
       ref={buttonRef as any}
-      accessibilityProps={{ ...makeAccessible({ role: 'button', label: accessibilityLabel }) }}
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+      // @ts-ignore: On React Native it will always be undefined but TS doesn't understand that
+      as={renderElement}
+      href={href}
+      target={target}
+      rel={rel}
```

Comment:
We auto-set rel based on href. Can we add that here?

Reply:
done